archeologies-et-sciences-de-lantiquite.csl

```diff
@@ -0,0 +1,700 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="fr-FR">
+  <info>
+    <title>Archéologies et Sciences de l’Antiquité</title>
```

Comment:
(@adam3smith, do you have any opinion whether we should put a hyphen after "l" in the style ID, for "l’Antiquité"? We're not consistent in the repository right now. Generated styles don't use a hyphen, while quite a few manually written styles do.

We're also not consistent about a preceding "d", as in e.g. "d'histoire". We do always forgo a hyphen for trailing "'s".)